pkg/db/db.go

```diff
@@ -431,6 +431,9 @@ func (h *DBHandler) DBReadEslEventLaterThan(ctx context.Context, tx *sql.Tx, esl
 }
 
 // RELEASES
+// Releases work a bit different, because they are already immutable.
+// We still store the eslId for consistency with other tables,
+// but technically it's not required here.
 
 type DBRelease struct {
```

Comment:
This type is unused (and kind of misleading). Let's delete it.

Reply:
done